lib/run_part.c

```diff
@@ -61,7 +61,7 @@ int run_parts (const char *directory, const char *name, const char *action)
 		struct stat  sb;
 
 		if (asprintf(&s, "%s/%s", directory, namelist[n]->d_name) == -1) {
-			fprintf(stderr, "could not allocate memory\n");
+			fprintf (shadow_logfd, "could not allocate memory\n");
```

Comment:
I would rather do the opposite white-space change.  The tendency is to get rid of it.